scipy/linalg/meson.build

```diff
@@ -10,16 +10,16 @@ cython_linalg = custom_target('cython_linalg',
     'cython_blas.pxd',
     'cython_lapack.pxd',
     '_blas_subroutines.h',
-    '_lapack_subroutines.h',
-    '_blas_subroutine_wrappers.f',
-    '_lapack_subroutine_wrappers.f'
+    '_lapack_subroutines.h'
   ],
   input: '_generate_pyx.py',
   command: [py3, '@INPUT@', '-o', '@OUTDIR@'],
   depend_files: [
     'cython_blas_signatures.txt',
     'cython_lapack_signatures.txt',
   ],
+  # Set PYTHONPATH to import from _build_utils/_wrappers_common.py
+  env: {'PYTHONPATH': meson.global_source_root() / 'scipy/_build_utils'},
```

Comment:
This I wasn't 100% sure about; should be fine but in case it gives an issue later we can avoid the problem by importing `_wrappers_common` with `importlib` like so: https://github.com/numpy/numpy/blob/64676baf423c3e1c53b37ae413c3ca6aaecc642a/numpy/_core/code_generators/genapi.py#L27-L32